### PR TITLE
add reindex api example

### PR DESCRIPTION
**Pull request by Yogendra0Sharma**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -583,24 +583,48 @@ <h3 class="ui header">Indices status and management</h3>
...
-
-
-
-
-
-
+	<a name="reIndexes"></a>
+	<h2 class="ui dividing header">Reindex API</h2>
+	<h3 class="ui header">Simple Reindex Operation</h3>
+	<pre class="es1 es2"><code>POST /_reindex
```
Comment (damienalexandre) [inline index.html]: The ReIndex API was not available in Elasticsearch 1, so the es1 class should not be set here. In fact the whole "reindex" part should be enclosed in a div with "es2 es5" classes.

File: index.html [hunk 2]
```diff
@@ -583,24 +583,48 @@ <h3 class="ui header">Indices status and management</h3>
...
+{
+  "source": {
+    "index": "test-index"
+  },
+  "dest": {
+    "index": "test-index-new"
+  }
+}</code></pre>
+	<pre class="es5"><code>POST /_reindex
```
Comment (damienalexandre) [inline index.html]: Same here, this example is compatible with es2 too.

**Comment by bastnic** [comment]
Thanks @Yogendra0Sharma !

**Review by damienalexandre — CHANGES_REQUESTED** [review]
Hi! Thanks a lot for your PR, here is just some minor changes to make.

File: index.html [hunk 3]
```diff
@@ -582,26 +582,51 @@ <h3 class="ui header">Indices status and management</h3>
...
+        "gender": "female"
+      }
+    }
+  },
+  "dest": {
+    "index": "test-index-new",
+    "type": "female"
+  }
+}
+}
```
Comment (damienalexandre) [inline index.html]: Looks like your JSON is not valid here?

File: index.html [hunk 4]
```diff
@@ -582,26 +582,51 @@ <h3 class="ui header">Indices status and management</h3>
...
+{
+  "source": {
+    "index": "test-index"
+  },
+  "dest": {
+    "index": "test-index-new"
+  }
+}</code></pre>
+	<h3 class="ui header">Selective Reindex Operation</h3>
+	<p >
```
Comment (damienalexandre) [inline index.html:607]: Can you remove the extra space?

File: index.html [hunk 5]
```diff
@@ -582,26 +582,51 @@ <h3 class="ui header">Indices status and management</h3>
...
-
-
-
-
-
-
-
-
+	<div class="es2 es5">
+	<a name="reIndexes"></a>
```
Comment (damienalexandre) [inline index.html]: As we do not add an entry in the menu, this can be removed too.

**Review by damienalexandre — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (Yogendra0Sharma) [reply index.html]: Yes, it's a typo. I have resolved this issue and submitted the change. please review.

**Comment by damienalexandre** [comment]
Thanks for the JSON typo, can you look at my other comments please?

File: index.html (hunk 5 above)
Comment (Yogendra0Sharma) [reply index.html]: do we have to add a reIndex option to the menu ? or just remove this code?

File: index.html (hunk 4 above)
Comment (Yogendra0Sharma) [reply index.html:607]: sure.

File: index.html (hunk 5 above)
Comment (damienalexandre) [reply index.html]: You should remove this option, we don't have enough space in the menu :grimacing:

**Comment by damienalexandre** [comment]
:white_check_mark: Hey, thank you @Yogendra0Sharma! :raised_hands:
That's some :sparkles: nice documentation :+1: and I'm glad to have the opportunity to merge it :pray:

**Pull request merged**